src/runmode-dpdk.c

```diff
@@ -1550,7 +1578,7 @@ static void *ParseDpdkConfigAndConfigureDevice(const char *iface)
     if (retval < 0) { // handles both configure attempts
         iconf->DerefFunc(iconf);
         if (rte_eal_cleanup() != 0)
-            FatalError("EAL cleanup failed: %s", strerror(-retval));
+            FatalError("EAL cleanup failed (err: %s)", strerror(-retval));
```

Comment:
I don't see this as an improvement. The old scheme is as clear, while being less verbose. Would rather unify on that. This also matches use of `strerror(errno)` elsewhere in the code

Reply:
same for the rest of the PR